work-products/metadata-spec/prov.md

```diff
@@ -0,0 +1,876 @@
+- [Sub-subsection](#sub-subsection)
+
+
+---
+
+
+# 1 Scope
+**Editor's Note:**
+Refer to template for OASIS style rules. 
+For readabilty, most were deleted prior to this point in doc.
+Some are left in after this point as guides on how to replace.
+
+
+What is the purpose and scope of this document?   
+Best practices: 
+
+
+- Expect this text to be reused in multiple other places to explain the specification in summary form.  
+- This is not the TC or OP scope (which is an IPR and rules boundary);  rather, this section is the summary intended purpose of this specification.  
+- Short is better; four paragraphs or less is recommended.  
+- If use of this spec is deliberately created to rely on or complement another standard, consider briefly mentioning that here as context.  
+- To the extent that discussion of the larger context of the spec, or its history, or the circumstances that led to its creation or revision, are necessary, they belong in the Introduction, not here. 
+
+
+---
+
+
+# 2 Definitions and Acronyms
+
+
+## 2.1 Definitions
+
+
+### 2.1.1 Terms Defined Elsewhere
+
+
+This document uses the following terms defined elsewhere:
+
+
+- Term 1: \[Reference\]: optional quoted definition.  
+- Term 2: \[Reference\]: optional quoted definition.
+- etc
+
+
+### 2.1.2 Terms Defined in this Document
+
+
+This document defines the following terms:
+
+
+- Term 1: some definition.  
+- Term 2: some definition.  
+- etc
+
+
+## 2.2 Abbreviations and Acronyms
+
+
+This document uses the following abbreviations and acronyms:
+
+
+- Term 1: expanded form.  
+- Term 2: expanded form.  
+- etc
+
+
+---
+
+
+# 3 Document Conventions
+
+
+## 3.1 Key Words
+
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**NOT RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in BCP 14 \[RFC2119\] \[RFC8174\] when, and only when, they appear in all capitals, as shown here.
+
+
+## 3.2 Typographical Conventions
+
+
+\< Describe any standards or typographical conventions that were followed when writing this document, such as fonts or highlighting that have special significance. If there are no typographical conventions than one is to put "None". \>
+
+
+---
+
+
+# 4 Introduction
+
+
+\< Any introductory text that is needed to explain this document \>
+
+
+## 4.1 Any Additional Introduction Subsections That are Needed
+
+
+\< Any needed text \>
+
+
+## 4.2 Changes From the Previous Version
+
+
+The list of changes from the previous version and any revision history can be found in Appendix 2\.
+
+
+---
+
+
+# 5 Provenance Information Model
+The information model of the provenance metadata is described in 
+human-readable property tables.
+The technical encoding may be found in section 6.
+
+The Data Provenance Standards are made up of three groups of metadata elements: 
+Source, Provenance, and Use.
```

Comment:
In general, I find the separation of the "trinity"-like provenance, source, and use a difficult concept. For example: why are provenance and source separate? 
From a reader perspective, I would in general push for lower case of many such English words with the general meaning and also order listings alphabetically.

Reply:
Personally I thought the sentence was greater than one line (at least how it shows up in my IDE) so I split it. But I'm fine with rejoining so I did.